### PR TITLE
fix: drop page counter styling

### LINKED ISSUE #6266: drop page counter styling
image
- there is slight gap between the last and the first one, so its not continuous infinite loop
- give it a black color, in dark mode white
image
- make it a bit faster, please 
- the time sometimes display "-" 
image
- currently on canary: https://canary.kodadot.xyz/stmn/unlockable

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #6266 
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed UI

image

### <samp>🤖 Generated by Copilot at e085ca7</samp>

Improved the UI of the `CountdownTimer.vue` component and added dark theme support to `styles/global.scss`. These changes enhance the user experience and accessibility of the NFT gallery.

### <samp>🤖 Generated by Copilot at e085ca7</samp>

> _The `CountdownTimer.vue` component got a makeover_
> _It looks better and works smoother than before_
> _No more negative numbers to show_
> _And the slides have a nice flow_
> _With colors that change with the `styles/global.scss` clover_

### CODE REVIEW

**Comment by exezbcz** [comment]
is it just me or the black colors on the drop page don't match :D 
image

- in light mode it should be k-black, black

can you please double check for my ocd 🥺

File: components/collection/unlockable/CountdownTimer.vue [hunk 1]
```diff
@@ -1,14 +1,15 @@
 <template>
-  <div ref="wrapper" class="px-4 py-4 keen-slider">
+  <div ref="wrapper" class="px-4 py-4 keen-slider background-inverse">
     <div
-      v-for="(section, index) in [1, 2, 3, 4]"
+      v-for="(section, index) in Array(10).fill(0)"
```
Comment (vikiival) [inline components/collection/unlockable/CountdownTimer.vue:4]: maybe good for `./const`?

File: components/collection/unlockable/CountdownTimer.vue [hunk 2]
```diff
@@ -20,7 +21,8 @@ import 'keen-slider/keen-slider.min.css'
 import { useKeenSlider } from 'keen-slider/vue.es'
 import { countDownTime } from './const'
 import { useCountDown } from './utils/useCountDown'
-const animation = { duration: 50000, easing: (t) => t }
+const animation = { duration: 30000, easing: (t) => t }
+const getPositiveNumber = (number: number) => (number >= 0 ? number : 0)
```
Comment (vikiival) [inline components/collection/unlockable/CountdownTimer.vue:25]:
```suggestion
const getPositiveNumber = (number: number) => Math.max(0, number)
```

**Comment by Jarsen136** [comment]
@exezbcz Color is Updated. Please check again

**Comment by exezbcz** [comment]
@Jarsen136 can you also please check mobile version 
image

**Comment by Jarsen136** [comment]
> @Jarsen136 can you also please check mobile version 

Please open the preview link on your mobile device or reload your browser. This component is not responsive for now while you adjust the browser window's width.

image

File: components/collection/unlockable/CountdownTimer.vue [hunk 3]
```diff
@@ -1,14 +1,15 @@
 <template>
-  <div ref="wrapper" class="px-4 py-4 keen-slider">
+  <div ref="wrapper" class="px-4 py-4 keen-slider background-inverse">
```
Comment (roiLeo) [inline components/collection/unlockable/CountdownTimer.vue:2]:
```suggestion
  <div ref="wrapper" class="p-4 keen-slider background-inverse">
```

File: components/collection/unlockable/CountdownTimer.vue [hunk 4]
```diff
@@ -1,14 +1,15 @@
...
     <div
-      v-for="(section, index) in [1, 2, 3, 4]"
+      v-for="(section, index) in Array(slidesCountOnTimeCountdown).fill(0)"
```
Comment (roiLeo) [inline components/collection/unlockable/CountdownTimer.vue:4]: I would make it computed value `Array(slidesCountOnTimeCountdown).fill(0)` 
`section` defined but not used

File: components/collection/unlockable/CountdownTimer.vue (hunk 3 above)
Comment (roiLeo) [inline components/collection/unlockable/CountdownTimer.vue:2]: I would create `has-background-inverse` like Bulma helpers

File: components/collection/unlockable/CountdownTimer.vue [hunk 5]
```diff
@@ -1,14 +1,15 @@
...
       class="keen-slider__slide is-flex no-wrap">
       <div class="no-wrap">
         <span class="mr-4 has-text-grey">Mint Phase Ends In</span
-        ><span
-          >{{ hours }} Hours : {{ minutes }} Minutes :
-          {{ seconds }} Seconds</span
+        ><span class="has-text-inverse"
+          >{{ getPositiveNumber(hours) }} Hours :
+          {{ getPositiveNumber(minutes) }} Minutes :
+          {{ getPositiveNumber(seconds) }} Seconds</span
```
Comment (roiLeo) [inline components/collection/unlockable/CountdownTimer.vue:12]:
```suggestion
          >{{ getPositiveNumber(hours) }} Hours:
          {{ getPositiveNumber(minutes) }} Minutes:
          {{ getPositiveNumber(seconds) }} Seconds</span
```

**Review by roiLeo — APPROVED** [review]
otherwise lgmt ✅

File: components/collection/unlockable/CountdownTimer.vue (hunk 5 above)
Comment (Jarsen136) [reply components/collection/unlockable/CountdownTimer.vue:12]: I think there is no need to change the space between the text.
image

File: components/collection/unlockable/CountdownTimer.vue [hunk 6]
```diff
@@ -24,6 +24,7 @@ import { useCountDown } from './utils/useCountDown'
 const animation = { duration: 30000, easing: (t) => t }
 const getPositiveNumber = (number: number) => Math.max(0, number)
 
+const slidesArray = Array(slidesCountOnTimeCountdown).fill(0)
```
Comment (roiLeo) [inline components/collection/unlockable/CountdownTimer.vue:27]:
```suggestion
const slidesArray = ref(Array(slidesCountOnTimeCountdown).fill(0))
```
?

Comment (Jarsen136) [reply components/collection/unlockable/CountdownTimer.vue:27]: Why should we make it a ref. it is a constant value.

Comment (roiLeo) [reply components/collection/unlockable/CountdownTimer.vue:27]: alright though it was common good pratice

**Comment by exezbcz** [comment]
@Jarsen136 yup, my bad. Thought I refreshed

**Review by vikiival — APPROVED** [review]

**Pull request merged**